webapp/pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>com.crc</groupId>
             <artifactId>commons</artifactId>
-            <version>v1.56.0</version>
+            <version>LOCAL-SNAPSHOT</version>
```

Comment:
I have to update this once https://github.com/ChildRescueCoalition/chat-lib-commons/pull/92 is merged